### PR TITLE
CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by benjaminwil**
- I added an initial CONTRIBUTING.md draft.
- I updated the GitHub icon so it could be used as an example in CONTRIBUTING.md.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - Include a descriptive `<title>` element for accessibility. For example, `<title>GitHub icon</title>`.
+  - Add `aria-labelledby="title"` to the `<svg>` element to improve screen reader support.
+  - Use a `viewBox` of 24 by 24, following [Google’s Material Design guidelines for system icons](https://material.io/guidelines/style/icons.html#icons-system-icons).
+  - Minify your SVG after exporting it. We recommend using [SVGO](https://github.com/svg/svgo) or [SVGOMG](https://jakearchibald.github.io/svgomg/) to automate this process.
+
+## Example
+
+Before minification, your SVG should look follow the template below:
+
+```
```
Comment (ericcornelissen) [inline CONTRIBUTING.md]: You can add "svg" here for syntax highlighting in the snippet 🙂

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<svg aria-labelledby="title" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+  <title>[BRAND] icon</title>
+  <path [VALUE] />
+</svg>
+```
+
+Here is the un-minified contents of the GitHub icon, for reference:
+
+```
```
Comment (ericcornelissen) [inline CONTRIBUTING.md]: You can add "svg" here for syntax highlighting in the snippet 🙂

File: _data/simple-icons.json [hunk 3]
```diff
@@ -726,6 +726,11 @@
             "source": "https://help.launchpad.net/logo/submissions"
         },
         {
+            "title": "Letterboxd",
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: I think its nicer to keep this separate, and only add it in #474 😉

**Comment by benjaminwil** [comment]
@ericcornelissen Thank you for your review.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+# Contributing
+
+This repository welcomes contributions and corrections. Before submitting a pull request, ensure that you respect these guidelines for SVGs:
+
+  - Include the SVG namespace declaration `xmlns="http://www.w3.org/2000/svg"`. This is not required if the SVG is included inline on HTML5 web pages.
+  - Include a descriptive `<title>` element for accessibility. For example, `<title>GitHub icon</title>`.
+  - Add `aria-labelledby="title"` to the `<svg>` element to improve screen reader support.
+  - Use a `viewBox` of 24 by 24, following [Google’s Material Design guidelines for system icons](https://material.io/guidelines/style/icons.html#icons-system-icons).
```
Comment (birjolaxew) [inline CONTRIBUTING.md]: Almost all existing SVG's use a `viewBox` of 16 by 16. The only ones that don't are HTML5 (14x16) and Nucleo (24x24). Why the change?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  - Minify your SVG after exporting it. We recommend using [SVGO](https://github.com/svg/svgo) or [SVGOMG](https://jakearchibald.github.io/svgomg/) to automate this process.
+
```
Comment (birjolaxew) [inline CONTRIBUTING.md]: Maybe add "remove as many attributes as possible - most SVGs should only have an `xmlns`, a `viewBox` attribute, and an `aria-labelledby` attribute" (or at least mention removing the `width` and `height` attributes).

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+# Contributing
+
+This repository welcomes contributions and corrections. Before submitting a pull request, ensure that you respect these guidelines for SVGs:
+
+  - Include the SVG namespace declaration `xmlns="http://www.w3.org/2000/svg"`. This is not required if the SVG is included inline on HTML5 web pages.
+  - Include a descriptive `<title>` element for accessibility. For example, `<title>GitHub icon</title>`.
+  - Add `aria-labelledby="title"` to the `<svg>` element to improve screen reader support.
```
Comment (birjolaxew) [inline CONTRIBUTING.md]: You're missing adding `role="img"` if you want to go all-out on accessibility (see [here](https://css-tricks.com/accessible-svgs/))

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  - Use a `viewBox` of 24 by 24, following [Google’s Material Design guidelines for system icons](https://material.io/guidelines/style/icons.html#icons-system-icons).
+  - Minify your SVG after exporting it. We recommend using [SVGO](https://github.com/svg/svgo) or [SVGOMG](https://jakearchibald.github.io/svgomg/) to automate this process.
+
+## JSON data for simpleicons.org
+
+In addition to following the guidelines for SVGs, please list new icons in the `_data/simple-icons.json` file. Each icon in the array has three required values:
```
Comment (birjolaxew) [inline CONTRIBUTING.md]: No "please" about it, you must do it or stuff will break :P

**Comment by benjaminwil** [comment]
@birjolaxew Thanks for the review. I made changes based on your feedback and brought up the issue of the `viewBox` size in #472.

**Comment by birjolaxew** [comment]
Nice work on this tho! If you have time you could write a small script to validate all of these rules (which could then be run by some CI on pull requests)

**Comment by birjolaxew** [comment]
Added the mentioned validation script in #486. Would be nice if you'd take a look @benjaminwil :)
